src/Mapbender/ManagerBundle/Resources/public/js/application-edit.js

```diff
@@ -324,6 +319,31 @@ $(function() {
         return false;
     });
 
+    function elementFormSubmit() {
+        var form = $("#elementForm"),
+            data = form.serialize(),
+            url = form.attr('action'),
+            self = this;
+
+        $.ajax({
+            url: url,
+            method: 'POST',
+            data: data,
+            error: function (e, statusCode, message) {
+                alert(Mapbender.trans("mb.application.save.failure.general") + ' ' + message);
+            },
+            success: function(data) {
+                if (data.length > 0) {
+                    $(form.parent()).html( data );
+                } else {
+                    $("#elementForm").data('dirty', false);
```

Comment:
you have access to the `form` variable in this scope. You don't have to query again.

```js
form.data('dirty', false);
```